internal/venice-common/src/main/proto/VeniceReadService.proto

```diff
@@ -1,11 +1,36 @@
 syntax = 'proto3';
 package com.linkedin.venice.protocols;
+import "google/protobuf/empty.proto";
 
 option java_multiple_files = true;
 
 service VeniceReadService {
-  rpc get (VeniceClientRequest) returns (VeniceServerResponse) {}
-  rpc batchGet(VeniceClientRequest) returns (VeniceServerResponse) {}
+
+  rpc get (VeniceClientRequest) returns (VeniceServerResponse) {
+    option deprecated = true;
+  }
+
+  rpc batchGet(VeniceClientRequest) returns (VeniceServerResponse) {
+    option deprecated = true;
+  }
+
+  rpc singleGet(SingleGetRequest) returns (SingleGetResponse) {}
+
+  rpc multiGet(MultiGetRequest) returns (MultiKeyResponse) {}
```

Comment:
Sorry for the confusion here... I know I am the one that asked for marking one of the APIs as deprecated, but I did not have the full understanding at the time I said that...

Since we are not yet implementing the new protocol buffers on the client-side, then effectively these old `get/batchGet` protocols are still the only valid one... the new ones are still experimental, they're not implemented end-to-end, and they might still change before becoming the new protocols we use (e.g. in case we decide to model the multi-key content as separate units).

So given all of the above, I think it's incorrect to mark the old protocols as deprecated, both here and in the Java code which use them.

Sorry again for the flip flopping...